localstack-core/localstack/utils/crypto.py

```diff
@@ -183,3 +196,106 @@ def decrypt(
     decrypted = decryptor.update(encrypted) + decryptor.finalize()
     decrypted = unpad(decrypted)
     return decrypted
+
+
+def pkcs7_envelope_encrypt(plaintext: bytes, recipient_pubkey) -> bytes:
```

Comment:
Parameter `recipient_pubkey` lacks type hint here, can you please add it?

Reply:
Yes, added.